Makefile

```diff
@@ -361,6 +362,7 @@ e2e-run:
 		--operator-image=$(OPERATOR_IMAGE) \
 		--e2e-image=$(E2E_IMG) \
 		--test-regex=$(TESTS_MATCH) \
+		--test-licence=$(TEST_LICENSE) \
```

Comment:
We need to settle on a spelling variant here :-(

Reply:
What prevents us from fixing it?

Reply:
Nothing. I think I am going for the US spelling `license` which is consistent with what Elasticsearch has https://www.elastic.co/guide/en/elasticsearch/reference/current/licensing-apis.html

Reply:
I'd go for license, but that's not a strong opinion.